### PR TITLE
fix: set cross_model to false instead of YAML null in agent defs

### PR DESCRIPTION
**Pull request by fredericboyer**
## Summary
- `cross_model:` frontmatter field in 6 reviewer agent definitions (knuth, szabo, brooks in both `templates/agents/` and `.dev-team/agents/`) was empty (YAML null due to inline comment only)
- Changed to explicit `cross_model: false` to prevent misconfiguration if a runtime ever reads this field
- Addresses Copilot feedback originally raised on closed PR #220

## Files changed
- `templates/agents/dev-team-knuth.md`
- `templates/agents/dev-team-szabo.md`
- `templates/agents/dev-team-brooks.md`
- `.dev-team/agents/dev-team-knuth.md`
- `.dev-team/agents/dev-team-szabo.md`
- `.dev-team/agents/dev-team-brooks.md`

## Test plan
- [x] `npm test` passes (308/308)
- [x] Verified YAML frontmatter parses correctly with explicit `false`

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: .dev-team/agents/dev-team-knuth.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: dev-team-knuth
 description: Quality auditor. Use to audit code for correctness gaps, missing test coverage, boundary conditions, and unproven assumptions. Read-only — identifies what is missing or unproven, does not write code or tests.
 tools: Read, Grep, Glob, Bash, Agent
 model: opus
-cross_model: # recommended when multi-model runtime available (ADR-023)
+cross_model: false # recommended when multi-model runtime available (ADR-023)
```
[the remark this answers is not in the document]
Comment (fredericboyer) [reply .dev-team/agents/dev-team-knuth.md]: Addressed in PR #229. Removed `cross_model: false` from all 6 reviewer agent definitions (3 in .dev-team/agents/, 3 in templates/agents/). Per ADR-023, the field should be a vendor-qualified string (e.g., `openai:o3`) — omitting it entirely is the correct way to indicate no cross-model assignment until runtime support is available.

File: templates/agents/dev-team-knuth.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: dev-team-knuth
 description: Quality auditor. Use to audit code for correctness gaps, missing test coverage, boundary conditions, and unproven assumptions. Read-only — identifies what is missing or unproven, does not write code or tests.
 tools: Read, Grep, Glob, Bash, Agent
 model: opus
-cross_model: # recommended when multi-model runtime available (ADR-023)
+cross_model: false # recommended when multi-model runtime available (ADR-023)
```
[the remark this answers is not in the document]
Comment (fredericboyer) [reply templates/agents/dev-team-knuth.md]: Addressed in PR #229. Removed `cross_model: false` — see reply on the dev-team-knuth.md thread for details.
